i18n/translations/ar.json

```diff
@@ -0,0 +1,3139 @@
+{
```

Comment:
can we remove the above .gitkeep file
<img width="433" alt="image" src="https://user-images.githubusercontent.com/56330681/187119800-71cacce0-2449-4397-b3be-49125dd2568b.png">

Reply:
Fixed, Thanks :pray: